src/rust/engine/fs/store/src/lib.rs

```diff
@@ -941,6 +949,7 @@ impl Store {
       .await?
       .walk(&mut |_, entry| match entry {
         directory::Entry::File(f) => file_digests.push(f.digest()),
+        directory::Entry::Symlink(_) => todo!(),
```

Comment:
Do nothing here. Symlinks are always present since there is no separate digest to load for them.